SigningService/Agents/SignAgent.cs

```diff
@@ -45,7 +46,7 @@ private void SignPackagePart(PackagePart packagePart)
         {
```

Comment:
Updating the package in-place is fine, but if that's what we're doing we need to clean up the associated method signatures. The Sign method should not return anything.